frontend/src/pages/projects/screens/detail/storage/ManageStorageModal.tsx

```diff
@@ -20,10 +20,9 @@ import useRelatedNotebooks, {
 import { getPvcDescription, getPvcDisplayName, getPvcTotalSize } from '~/pages/projects/utils';
 import NotebookRestartAlert from '~/pages/projects/components/NotebookRestartAlert';
 import useWillNotebooksRestart from '~/pages/projects/notebook/useWillNotebooksRestart';
+import DashboardModalFooter from '~/concepts/dashboard/DashboardModalFooter';
 import ExistingConnectedNotebooks from './ExistingConnectedNotebooks';
 
-import './ManageStorageModal.scss';
```

Comment:
It's odd to me that there is a generic style in ManageStorageModal.scss and this file isn't even used by ManageStorageModal.tsx. It would make more sense to move those styles to a more generic location.

Including these styles in App.scss or a new overrides.scss file makes more sense. cc @andrewballantyne